beacon-chain/powchain/service.go

```diff
@@ -281,8 +281,8 @@ func (s *Service) ChainStartEth1Data() *ethpb.Eth1Data {
 
 // PreGenesisState returns a state that contains
 // pre-chainstart deposits.
-func (s *Service) PreGenesisState() state.BeaconState {
-	return s.preGenesisState
+func (s *Service) PreGenesisState() (state.BeaconState, error) {
+	return s.preGenesisState, nil
```

Comment:
Why this change if we never return an error?

Reply:
This service implements an interface. Another implementation needs to return an error.
https://github.com/prysmaticlabs/prysm/blob/8a10f4e87e076fc50b06fc24efbf31a6dffe6c95/beacon-chain/deterministic-genesis/service.go#L135-L141